packages/compass-generative-ai/src/components/generative-ai-input.tsx

```diff
@@ -341,24 +397,38 @@ function GenerativeAIInput({
       <div className={inputBarContainerStyles}>
         <div className={inputContainerStyles}>
           <div className={isFetching ? gradientAnimationStyles : undefined}>
-            <div
-              className={contentWrapperStyles}
-              data-testid="ai-user-text-input-wrapper"
-            >
-              <TextInput
+            <div className={contentWrapperStyles}>
+              <button
```

Comment:
drive-by: moved this button up so that it's earlier in the tab order which is more inline with where it's rendered.